sim/midas/src/main/scala/midas/widgets/Assert.scala

```diff
@@ -22,23 +22,29 @@ class AssertBridgeModule(assertMessages: Seq[String])(implicit p: Parameters) ex
     val hPort = IO(HostPort(Input(UInt(numAsserts.W))))
     val resume = WireInit(false.B)
     val cycles = RegInit(0.U(64.W))
-    val asserts = hPort.hBits
+    val q = Module(new Queue(hPort.hBits.cloneType,2))
```

Comment:
Nit: Space between comma and additional parameters.

Reply:
Added space in fixup commit.